src/import/importd.c

```diff
@@ -1351,6 +1357,26 @@ static int manager_run(Manager *m) {
                         m);
 }
 
+static void parse_env(Manager *m) {
+	int r;
+
```

Comment:
`assert(m)`

Reply:
Indeed. Also fixed indentation -- apparently my editor didn't pull the EditorConfig file.